components/flipper.py

```diff
@@ -2,18 +2,28 @@
 
 
 class Flipper:
+    #this flips a bucket with an arm that has a spinning motor
```

Comment:
We can change this from a comment into a Python [docstring](https://www.python.org/dev/peps/pep-0257/), like so: `""" This flips a bucket with an arm that has a spinning motor"""`